### PR TITLE
Koko Bars

### PR DESCRIPTION
**Pull request by LadyFowls**
- rscadd: "Add's KoKo bars, a unathi themed candy."

### CODE REVIEW

**Comment by LadyFowls** [comment]
!review

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 1]
```diff
@@ -1207,6 +1207,36 @@
 	if (prob(dose))
 		M.vomit()
 
+
+/datum/reagent/mental/kokoreed
+	name = "KoKo Reed Juice"
+	id = "kokoreed"
+	description = "A tall, green, grass-like plant grows up to 7 feet tall. In the past it thrived in the wet marshlands outside Iz'iz, having evolved to take nutrients from the otherwise toxic byproducts of the algae-contaminated waters. It was discovered by the Unathi colonists that the stalk can be chewed to induce a soothing sense of warmth and comfort, but only to Unathi due to its unique interactions with Unathi physiology. Other species simply taste a sweet, stringy reed. ."
```
Comment (alsoandanswer) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Grammar.

**Comment by alsoandanswer** [comment]
wtf

**Comment by alsoandanswer** [comment]
kokobar

also, please use these sprites

**Comment by Geevies** [comment]
If appears I was mistaken, I will give this a proper review tomorrow.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 2]
```diff
@@ -1207,6 +1207,36 @@
 	if (prob(dose))
 		M.vomit()
 
+
+/datum/reagent/mental/kokoreed
+	name = "KoKo Reed Juice"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: is it KoKo or Koko, there are 3 different instances of it being spelled differently

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm (hunk 1 above)
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: This description is way too long dude. Its a candy bar reagent for god's sake.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm [hunk 3]
```diff
@@ -1207,6 +1207,36 @@
...
+	id = "kokoreed"
+	description = "A tall, green, grass-like plant grows up to 7 feet tall. In the past it thrived in the wet marshlands outside Iz'iz, having evolved to take nutrients from the otherwise toxic byproducts of the algae-contaminated waters. It was discovered by the Unathi colonists that the stalk can be chewed to induce a soothing sense of warmth and comfort, but only to Unathi due to its unique interactions with Unathi physiology. Other species simply taste a sweet, stringy reed. ."
+	reagent_state = LIQUID
+	color = "#008000"
+	metabolism = 0.0016 * REM
+	overdose = 3
+	data = 0
+	taste_description = "sugar"
+	goodmessage = list("You feel pleasantly warm.","You feel like you've been basking in the sun.","You feel focused and warm..")
```
Comment (NonQueueingMatt) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]:
```suggestion
	goodmessage = list("You feel pleasantly warm.","You feel like you've been basking in the sun.","You feel focused and warm...")
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -511,6 +511,22 @@
 	. = ..()
 	reagents.add_reagent("sugar", 3)
 
+
+/obj/item/weapon/reagent_containers/food/snacks/candy/koko
+	name = "koko bar"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks.dm]:
```suggestion
	name = "\improper Koko bar"
```

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm (hunk 1 above)
Comment (Alberyk) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm]: Extra lore can be moved to description_fluff

**Pull request merged**